nbcm.js

```diff
@@ -1,3 +1,6 @@
+const openDelay = 500; // Menu opening delay in milliseconds
+let openTimeout;
```

Comment:
Crap I should not have left this line in. I'll resubmits if you would like but I'll wait to see if you have any other comments.